### PR TITLE
Add some documentation

### PR DESCRIPTION
**Pull request by nullromo**
There was an undocumented option in the config, so I added an explanation for it.

It was also not clear that the default mapping includes both <kbd>\`</kbd> _and_ <kbd>'</kbd>, so I added a note for that.

### CODE REVIEW

**Comment by winston0410** [comment]
Hi @nullromo , thank you for your commit. After I merge your other PRs, it has a commit conflict. Do you mind resolving it yourself?

Btw as I am occupied now, do you want to be the maintainer for this project? I can transfer this project to you.

**Comment by nullromo** [comment]
@winston0410 Thanks for merging those! I fixed the merge conflict in the PR as well.

Sure, I would love to take on the responsibility.

**Review by winston0410 — APPROVED** [review]

**Pull request merged**
